server/src/main/java/org/opensearch/common/cache/stats/CacheStats.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.cache.stats;
+
+/**
+ * Interface for any cache specific stats.
+ * TODO: Add rest of stats like hits/misses.
+ */
+public interface CacheStats {
+    // Provides the number of entries in cache.
+    long count();
```

Comment:
This could be the total, or the current count, might want to be verbose.

Reply:
This is the current count, will change.

Reply:
I mean by changing the variable name, eg. `currentCount`